### ISSUE TITLE
Bookmark icon (edit mode) - not accessible with screen reader braille

### ISSUE THREAD

**Issue by CarmenHilpert** [ckeditor/ckeditor5#19160]
## 📝 Provide detailed reproduction steps (if any)

1. Open https://ckeditor.com/docs/ckeditor5/latest/features/bookmarks.html
2. Place cursor before bookmark icon
4. Press the arrow right key

### ✔️ Expected result

- The screen reader braille should ouput the bookmark icon or an alternative text.

### ❌ Actual result

- The Screen reader braille output "et"

### ❓ Possible solution

- The bookmark icon is a decorative graphic and should stay hidden as it is. 
- Insert a text for screen readers only, e.g.
`   <span aria-label="Bookmark <name of bookmark>" class="sr-only"></span>`

## 📃 Other details

- The div with class="ck-fake-selection-container" is not linked to the bookmark, but the screen reader accounces the information.
* Browser: Edge 138.0.3351.65
* OS: Windows 11 (10.0.22631 Build 22631)
* Screenrader: JAWS 2024
* First affected CKEditor version: not shure when it started - seen in v44.3 and later
* Installed CKEditor plugins: see demo page

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
